app/src/components/Preferences/PreferencesCard.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  Typography,
+  LocationOnIcon,
+} from '../../store/index';
+import { useStyles } from './PreferencesCardStyles';
+
+export default function PreferencesCard() {
+  const classes = useStyles();
+  return (
+    <Card className={classes.root}>
+      <CardContent>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}>
+          <Typography variant='h4'>Training</Typography>
+          <Typography variant='h6' className={classes.subBody}>
+            Edit
+          </Typography>
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            paddingTop: '5px',
+            gap: '5px',
```

Comment:
What does `gap` do? Just curious.

Reply:
Gap creates the space between the flex items.

Reply:
Cool, never heard of it!